Configuration/NodeTypes.yaml

```diff
@@ -3,69 +3,64 @@
     fulltext:
       enable: true
   properties:
-    '__identifier':
+    'neos.node_identifier':
```

Comment:
is it on purpose you use the dot notation here? I'd assume the snake_case to be used.

Reply:
Nope, was an experiment to use a "neos" object with subfields. But that seems to need a serious refactoring of Flowpack.Elasticsearch which I don't want to do this time